swift-sdk/ITBConsts.swift

```diff
@@ -156,6 +156,7 @@ public extension AnyHashable {
     public static let ITBL_IN_APP_HTML = "html"
     public static let ITBL_IN_APP_HREF = "href"
     public static let ITBL_IN_APP_DISPLAY_SETTINGS = "inAppDisplaySettings"
-    public static let ITBL_IN_APP_PAYLOAD = "payload"
+    public static let ITBL_IN_APP_CUSTOM_PAYLOAD = "customPayload"
+    public static let ITBL_IN_APP_CHANNEL_NAME = "channelName"
```

Comment:
Not related to this PR, but let's remove/hide `channelName` before release; we'll probably end up not using it anyway.